token-holders/how-to-stake.md

```diff
@@ -0,0 +1,40 @@
+# The staking UI
+
+{% hint style="info" %}
+Staked tokens can't be withdrawn immediately, make sure you understand the [rules](staking-rules.md) before staking your tokens.
+{% endhint %}
+
+First, go to the [staking frontend](https://staking.pyth.network/) and connect your wallet
```

Comment:
staking frontend seems to dev-catered, maybe staking app is a more user-friendly term?

Reply:
Thanks